src/kibana/components/agg_types/controls/values.html

```diff
@@ -0,0 +1,29 @@
+<div class="form-group" ng-controller="agg.type.params.byName.values.controller">
+  <label>Values</label>
+  <div
+    ng-repeat="value in agg.params.values track by $index"
+    class="form-group vis-editor-agg-form-row vis-editor-agg-form-row">
+
+    <input
+      ng-model="agg.params.values[$index]"
+      values-list="agg.params.values"
+      input-focus
+      class="form-control">
+
+    <button type="button" ng-click="remove($index, 1)" class="btn btn-danger btn-xs">
+      <i class="fa fa-times"></i>
+    </button>
+  </div>
+
+  <input ng-model="validLength" name="validLength" required type="hidden">
+  <p ng-show="aggForm.validLength.$invalid" class="text-danger text-center">
+    You must specify at least one value
+  </p>
+
+  <button
+    ng-click="add()"
+    type="button"
+    class="sidebar-item-button primary">
+    <i class="fa fa-plus"></i> Add value
+  </button>
+</div>
```

Comment:
I tried to do this in the beginning, but it turned out more complex that having two files represented, in fact, different views. If you have a good idea how to reuse the same view across different controllers, please let me know, I'd be happy to make this change.